packages/extension-ui/src/components/ValidatedInput.tsx

```diff
@@ -20,17 +23,22 @@ type Props<T extends BasicProps> = T & {
   defaultValue?: string;
   onValidatedChange: (value: string) => void;
   validator: Validator<string>;
+  isCapsLockChecked?: boolean;
 };
 
 function ValidatedInput<T extends Record<string, unknown>>({
   className,
   component: Input,
   defaultValue,
+  isCapsLockChecked = false,
```

Comment:
The name had me thinking that this indicates whether the capslock has already been checked. My suggestion is sth like:
```suggestion
  shouldCheckCapsLock = false,
```